shell/app/modules/project/common/components/issue/subscribers-selector.tsx

```diff
@@ -0,0 +1,195 @@
+    setUsersMap({ ..._userMap });
+  }, []);
+
+  React.useEffect(() => {
+    setSubscribers(subscribersProps || []);
+  }, [subscribersProps]);
+
+  const menu = (
+    <Menu>
+      <Menu.Item>
+        {isFollowed ? (
+          <div
+            onClick={async () => {
+              if (issueID) {
+                await unsubscribe({ id: issueID });
+                getIssueDetail({ id: issueID, type: issueType });
+                getIssueStreams({ type: issueType, id: issueID, pageNo: 1, pageSize: 50 });
+              } else {
+                let index = subscribers.findIndex((item) => item === loginUserId);
+                subscribers.splice(index, 1);
+                setSubscribers([...subscribers]);
+              }
+            }}
+          >
+            <IconPreviewCloseOne />
+            取消关注
+          </div>
+        ) : (
+          <div
+            onClick={async () => {
+              if (issueID) {
+                await subscribe({ id: issueID });
+                getIssueDetail({ id: issueID, type: issueType });
+                getIssueStreams({ type: issueType, id: issueID, pageNo: 1, pageSize: 50 });
+              } else {
+                setSubscribers([...subscribers, loginUserId]);
+              }
+            }}
+          >
+            <IconPreviewOpen />
+            关注
+          </div>
+        )}
+      </Menu.Item>
+      <Menu.Item>
+        <MemberSelector
+          scopeType="project"
+          className="issue-member-select"
+          dropdownClassName="issue-member-select-dropdown"
+          scopeId={projectId}
+          allowClear={false}
+          ref={memberRef}
+          mode="multiple"
+          value={subscribers || []}
+          onChange={async (ids: Array<number | string>, options: Array<{ userId: string }>) => {
+            if (issueID) {
+              await batchSubscribe({ id: issueID, subscribers: ids });
+              getIssueDetail({ id: issueID, type: issueType });
+              getIssueStreams({ type: issueType, id: issueID, pageNo: 1, pageSize: 50 });
+            } else {
+              let newUsers = options.filter((item) => !usersMap[item.userId]);
+              newUsers.forEach((item) => {
+                usersMap[item.userId] = item;
+              });
+              setUsersMap({ ...usersMap });
+              setSubscribers(ids);
+            }
+          }}
+          resultsRender={() => (
+            <span
+              onClick={(e) => {
+                e.stopPropagation();
+                memberRef.current?.show();
+              }}
+            >
+              <IconPlus />
+              添加关注人
+              <IconRight className="add-follower-btn" />
+            </span>
+          )}
+        />
+      </Menu.Item>
+      <Menu.Divider />
+      <Menu.Item>
+        <div onClick={(e) => e.stopPropagation()}>
+          <div className="followers-num">{subscribers ? `${subscribers.length}位成员正在关注` : '暂无成员关注'}</div>
```

Comment:
useless condition, subscribers is always an array. condition will always be true